src/main/java/cz/cvut/kbss/termit/dto/readonly/ReadOnlyTerm.java

```diff
@@ -43,13 +43,16 @@
     @OWLObjectProperty(iri = SKOS.RELATED_MATCH, fetch = FetchType.EAGER)
     private Set<TermInfo> relatedMatch;
 
+    @Properties
+    private Map<String,Set<String>> properties;
+
     @Types
     private Set<String> types;
 
     public ReadOnlyTerm() {
     }
 
-    public ReadOnlyTerm(Term term) {
+    public ReadOnlyTerm(final Term term, final Set<String> props) {
```

Comment:
I'd rename it to something like `propertiesToExport`. `props` tells nothing of the content and purpose of the parameter.

Reply:
Also, given the number of calls with an empty set (mainly in tests), I'd vote for creating another constructor taking just the `term`. This would greatly simplify the tests and remove some of the introduced unnecessary changes (e.g., in `ReadOnlyTermControllerTest` and `ReadOnlyTermServiceTest`).